.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,10 @@
+- id: jupytext
+  name: jupytext
+  description: Convert all notebooks into another format
```

Comment:
This description isn't quite right if you want to the opposite to occur, (generate clean ipynb from python files).

Reply:
Agreed, suggestions welcome